### ISSUE TITLE
Improve accessibility for sidebars

### ISSUE THREAD

**Issue by mirka** [Automattic/simplenote-electron#979]
The NavigationBar and NoteInfo sidebars should basically act like a modal, in that:

1. <kbd>Esc</kbd> should close it
1. Keyboard focus should jump to the first focusable element when the sidebar is activated
1. Keyboard focus should be trapped inside the sidebar
1. Keyboard focus should return to the original element when the sidebar is closed

We should also consider `tablist` / `tabpanel` semantics for the NavigationBar so it makes sense in screen readers.

**Comment by belcherj** [comment]
Note info will be a modal soon so no need to do that one.
